repo/packages/K/kubernetes/1/marathon.json

```diff
@@ -0,0 +1,91 @@
+{
+  "id": "{{kubernetes.app-id}}",
```

Comment:
Can you take a look at https://github.com/sttts/universe/pull/1? The framework-name is then actually becoming the framework-id, right?